src/controllers/img.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+exports.get=(req,res)=>{
+  res.render("home",{})
+};
+
+exports.post=(req,res)=>{
```

Comment:
I think the post requests can go on another pull request once you've got something to test it with!